src/functionalTest/java/uk/gov/hmcts/reform/laubackend/idam/serenityfunctionaltests/utils/TestConstants.java

```diff
@@ -6,8 +6,8 @@ public final class TestConstants {
     public static final String S2S_URL = "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal/testing-support";
 
     /*Logon endPoint*/
-    public static final String LOGON_ENDPOINT = "/audit/logon";
-    public static final String LOGON_DELETE_ENDPOINT = "/audit/logon/deleteAuditLogonRecord";
+    public static final String LOGON_ENDPOINT = "/audit/logon/";
```

Comment:
When you remove the slash from the path in the controller this change will be redundant.